.eslintrc.json

```diff
@@ -4,15 +4,27 @@
     "next",
     "next/core-web-vitals",
     "prettier",
-    "plugin:storybook/recommended"
+    "plugin:storybook/recommended",
+    "plugin:@typescript-eslint/recommended"
   ],
+  "parser": "@typescript-eslint/parser",
   "overrides": [
+    {
+      "files": ["**/*.mjs"],
+      "rules": {
+        "@typescript-eslint/no-empty-function": 1
+      }
+    },
     {
       "files": ["**/*.ts?(x)"],
       "plugins": ["@typescript-eslint", "no-relative-import-paths"],
       "rules": {
         "@typescript-eslint/consistent-type-imports": "error",
         "@typescript-eslint/no-unused-vars": "error",
+        "@typescript-eslint/no-non-null-assertion": 1,
```

Comment:
Could you comment on why you added these rules? 🤔

Reply:
These rules are present in the `"plugin:@typescript-eslint/recommended"` but they are specified with value `2`, thus they throw linting error when linter executes, I overwrote the value to be `1` to change the `error` property to `warning`.

May I leave it like that or change it to `2`/remove it?

Reply:
Why would you change the default behaviour? If they throw errors, they should throw errors 😅 

And then the errors should be fixed...

Reply:
Yes, right.
But existing code is getting affected now, may I still leave the default behaviour?

Reply:
In other words, if ESlint starts to give errors, then we should fix the Errors 👀 if you need guidance, just let us know :)

Reply:
<img width="1440" alt="Screenshot 2023-07-28 at 15 50 26" src="https://github.com/nodejs/nodejs.org/assets/61611599/95804fce-ecfa-4e45-8124-a4feaa26879e">

See this, these are the errors!

Reply:
May I go ahead and fix these?

Reply:
Pushed new changes, can you please review them?

Reply:
Will do :)

Reply:
Okay, thank you!
Tell me if there's any change needed, will make that change!